DEHPCommon/HubController/HubController.cs

```diff
@@ -392,39 +392,92 @@ public IEnumerable<NestedElement> GetNestedElementTree(Option option, DomainOfEx
         /// <param name="iteration"></param>
         /// <param name="domain"></param>
         public async Task Upload(File file = null, Iteration iteration = null, DomainOfExpertise domain = null)
+        {
+            await Upload(null, file, iteration, domain);
+        }
+
+        /// <summary>
+        /// Upload one file to the <see cref="DomainFileStore"/> of the specified domain or of the active domain
+        /// </summary>
+        /// <param name="filePath"></param>
+        /// <param name="file"></param>
+        /// <param name="iteration"></param>
+        /// <param name="domain"></param>
+        public async Task Upload(string filePath, File file = null, Iteration iteration = null, DomainOfExpertise domain = null)
```

Comment:
If filePath can be null, you could also assign it a default value, like the "file" parameter and the other parameters, Would it make the usage of the method clearer? Meaning that a filepath is not required

Reply:
I put in that way to maintain previous tests calling `Upload()`:
* With no parameters
* Only `File`
* Only `FileRevision`

I looks cleaner adding a default value, having only one `Upload()` entry.